rollup-boot.config.mjs

```diff
@@ -0,0 +1,69 @@
+import { readFileSync } from 'fs';
+
+import { babel } from '@rollup/plugin-babel';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+import json from '@rollup/plugin-json';
+import replace from '@rollup/plugin-replace';
+import { terser } from 'rollup-plugin-terser';
+
+const isProd = process.env.NODE_ENV === 'production';
+const prodPlugins = [];
+if (isProd) {
+  prodPlugins.push(terser());
+}
+
+const { version } = JSON.parse(readFileSync('./package.json').toString());
+
+// URL template which is expanded by the boot script. See `src/boot/url-template.js`.
+const localhost = '{current_scheme}://{current_host}';
+
+const notebookAppUrl = process.env.NOTEBOOK_APP_URL
+  ? `${process.env.NOTEBOOK_APP_URL}`
```

Comment:
Is this to ensure the value is a string?

Reply:
I think I copied this from existing code. The string interpolation looks superfluous here though.